Index: pages/FULLRESYNC.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>- What\n\t- Redis主从复制时，如果是从库和主库间的第一次复制，主库对从库的psync命令的回复；表示全量回复，即主库会把当前的所有数据都回复给\n- Why\n- How\n- How Good\n- Refs\n- See Also
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/pages/FULLRESYNC.md b/pages/FULLRESYNC.md
--- a/pages/FULLRESYNC.md	(revision 60dd599015461ec73bcc9b1786242998ece4d613)
+++ b/pages/FULLRESYNC.md	(date 1718164521102)
@@ -1,5 +1,5 @@
 - What
-	- Redis主从复制时，如果是从库和主库间的第一次复制，主库对从库的psync命令的回复；表示全量回复，即主库会把当前的所有数据都回复给
+	- Redis主从复制时，如果是从库和主库间的第一次复制，主库对从库的psync命令的回复；表示全量回复，即主库会把当前的所有数据都复制给从库
 - Why
 - How
 - How Good
